orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandlerTest.kt

```diff
@@ -589,10 +589,10 @@ object RunTaskHandlerTest : SubjectSpek<RunTaskHandler>({
             )
           }
 
-          it("attaches the exception to the stage context") {
+          it("attaches the exception to the taskExceptionDetails") {
             verify(repository).storeStage(
               check {
-                assertThat(it.context["exception"]).isEqualTo(exceptionDetails)
+                assertThat(it.tasks[0].taskExceptionDetails["exception"]).isEqualTo(exceptionDetails)
```

Comment:
Same question as above - does Deck not rely on this?

I'm also concerned about people who may be using SpEL on this field today which this would break.

Reply:
Made changes in `completeStageHandler` to add the task exception errors to the `stageContext` Exception errors in order to display in UI only to work in case where the status is FAILED_CONTINUE.The other fields in the `stageContext `exception remain untouched